src/borg/archiver.py

```diff
@@ -2857,7 +2857,7 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                                help='print statistics for the created archive')
         subparser.add_argument('--list', dest='output_list', action='store_true',
                                help='output verbose list of items (files, dirs, ...)')
-        subparser.add_argument('--filter', metavar='STATUSCHARS', dest='output_filter',
+        subparser.add_argument('--filter', metavar='AMUEdbchsfi-x', dest='output_filter',
```

Comment:
Or should I consolidate both texts, like with a dict with each status char & its meaning that could be pretty printed in the help text or displayed in short form with `"".join(statuschars.keys())`?